v2/orders/order-workflows-association.rst

```diff
@@ -0,0 +1,9 @@
+Associating an Order Type with a Workflow
+=========================================
+
+An order type can be associated with a specific workflow, and that forces the order to follow that workflow's rules such as to only move through the defined transitions. To associate an order type with an order workflow, go to ``/admin/commerce/config/order-types`` and select to Edit the desired order type. You can then choose the desired workflow from the Workflow dropdown field. Save the form.
+
+.. figure:: images/order_workflow_association.jpg
+   :alt: Associating an Order Type with an Order Workflow
+
+   Associating an Order Type with an Order Workflow
```

Comment:
I would like to see a section added here on how to create a workflow that will show up here. Code examples and high overviews welcome.